Vagrantfile

```diff
@@ -106,6 +106,7 @@ Vagrant.configure("2") do |config|
       # Only execute once the Ansible provisioner,
       # when all the machines are up and ready.
       if i == $num_instances
+        config.vm.provision :shell, inline: '[[ -x "$(command -v yum)" ]] && ifup eth1', privileged: true
```

Comment:
ip link set eth1 up works fine on CentOS \ Ubuntu. I am not sure about coreos. However, I think it's not a job of vagrant to bring enterfaces. The image should be created in the way where all interfaces are up.